changelogs/unreleased/6022-use-python-311.yml

```
@@ -0,0 +1,4 @@
+description: support python3.11
+issue-nr: 6022
+change-type: major
```

```suggestion
change-type: patch
```

Had missed this before. This is not a major. Bumping the actual minimal version might be but we're not there yet.